### ISSUE TITLE
default random seed shouldn't set to 1 if you train in `for seed in args.seed`

### ISSUE THREAD

**Issue by Eurus-Holmes** [Wuyxin/DIR-GNN#3]
When you use `default seed = 1` to train in `for seed in args.seed`, you only put one value to `all_info`, then you will get `nan` when calculate `torch.tensor(all_info['...']).std()` because in `tensor.std` is divided by `n-1`
<img width="860" alt="Screen Shot 2022-05-10 at 00 57 45" src="https://user-images.githubusercontent.com/34226570/167578474-c9152ca1-fe7d-4114-b184-cd3a5dce6fb2.png">

**Comment by Wuyxin** [comment]
Thanks, I changed `args.seed` as multi-seed by default. And you can just ignore ±xxx if you use single seed.
